scenarios/plugin.schema.json

```diff
@@ -2253,6 +2253,166 @@
 					"id",
 					"config"
 				]
+			},
+			{
+				"type": "object",
+				"title": "pod_egress_shaping Arcaflow scenarios",
+				"properties": {
+					"id": {
+						"type": "string",
+						"const": "pod_egress_shaping"
```

Comment:
should this be pod_network_shaping? Getting error it doesnt exist in the scenario file

Reply:
Hello @paigerube14 it should be "pod_egress_shaping" and vscode is able to populate the required text.
`- id: pod_egress_shaping
  config:
    namespace: 
    test_duration: 90
    wait_duration: 300
    instance_count: 1
    execution_type: parallel`

Reply:
Getting error 

"Value must be "pod_network_shaping".yaml-schema: pod_network_shaping Arcaflow scenarios(1)"